### DISCUSSION TITLE
How to install web version only in V9 using NPM

### DISCUSSION THREAD

**Discussion by joeshub** [pmndrs/react-spring#1016] [Support] [answered]
Hi,

I'm using **v9 rc3**, and using **npm**, not yarn. 

Is it possible to install _only_ the web version of react-spring and its required sub-projects?

Cheers

**Comment by aleclarson** [comment, marked as answer]
`@react-spring/web` is what you're looking for :)

**Reply by joeshub** [reply]
Awesome thank you!
